src/app/dashboard/dashboard.component.ts

```diff
@@ -7,9 +11,61 @@ import { Component, OnInit } from '@angular/core';
 })
 export class DashboardComponent implements OnInit {
 
-  constructor() { }
+  projects: Project[];
+
+  constructor(private projectService: ProjectService) { }
 
   ngOnInit() {
+    this.getProjects();
+  }
+
+  getProjects(): void {
+    this.projectService.getProjects()
+      .subscribe(projects => this.projects = projects);
+  }
+
+  calculateStatusClasses(lastBuild) {
+    const status = lastBuild.worker.status;
+
+    switch (status) {
+      case 'Success':
+        return this.successStateClasses();
+      case 'Failed':
+        return this.failureStateClasses();
+      case 'Active':
+        return this.activeStateClasses();
+      default:
+        return this.unknownStateClasses();
```

Comment:
The class calc function itself. It determines what CSS classes to apply to each project status icon, e.g. "passed", "failed", "active", "unknown. Not very elegant, as requires passing in a lastBuild object, fetching the worker status, and comparing the status strings. I wonder if we could simplify the data model on the Brigade API. It'd be nice to replace this switch with object literals, using the status value to construct the appropriate css class name, e.g. `project-failed`. Still need to apply a corresponding icon, however.

cc @technosophos